js/controller.js

```diff
@@ -412,6 +414,10 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
 
       this.state.hideMultiAudioIcon = !!params.hideMultiAudioIcon;
 
+      if (this.state.playerParam.chromecast && this.state.playerParam.chromecast.enable && this.state.playerParam.chromecast.enable == true) {
```

Comment:
Not sure about the assignment here. This makes the player param looks like:
var param = {
  chromecast.enable = true
};

Should we simplify it to be:
var params = {
  chromecast: true
}

Reply:
I'm okay with either one.

Reply:
Thanks for confirming. Before we make any decision, I found that, even when you do not specify appID, the cast button still appear and you are able to cast. I think we need to disable chromecast IF:

chromecast.appID is not provided, and/OR if chromecast.enable is set to FALSE. By default, chromecast.enable is FALSE.
chromecast.appID should not have a default appID.

Reply:
@aldigun Yeah, if the appId it's nor provided the core.js it's using our Prod AppID by default...Ok, I will work on that changes